app/assets/javascripts/components/TaskList.jsx

```diff
@@ -0,0 +1,24 @@
+class TaskList extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			tasks: [],
+		};
+	}
+
+	componentDidMount() {
+		Requester.get(`/api/users/${this.props.userId}/tasks`).then((tasks) => {
+			this.setState({ tasks });
+		});
+	}
+
+  render() {
+  	const tasks = this.state.tasks;
+  	const taskList = tasks && tasks.map((task, index) => <TaskItem task={task} key={index} />);
```

Comment:
I don't think the tasks null check is necessary since it's guaranteed to be at least an empty array in the constructor